### ISSUE TITLE
Can’t do the cron job for backup gcp

### ISSUE THREAD

**Issue by hvalentino** [jacksegal/google-compute-snapshot#91]
Hi Jack, first of all thank you for your script! I wonder if you can help with this... when I tried to run the job, it gives me, no disks were found please check your script options/account permission. Do you think you can help me with this? Thanks again !!

**Comment by jacksegal** [comment]
Hi @hvalentino 

Does the script work when you run it manually?  

If so, have you tried using `sudo` in the CRON command? 

Using the example in the README that would be:

```
0 5 * * * sudo root /opt/google-compute-snapshot/gcloud-snapshot.sh > /dev/null 2>&1
```

Another user has [reported](https://github.com/jacksegal/google-compute-snapshot/issues/90) that this fixed the problem for them!

Thanks,
Jack

**Comment by hugokx** [comment]
Hi all,

So I started the thread regarding "command not found" only when the script was run through crontab.

To resume what happened

During normal shell session in terminal

gcloud  --> command is found
sudo gcloud --> command not found

Once the script is run through crontab with current user (NOT sudo)
`0 5 * * * /opt/google-compute-snapshot/gcloud-snapshot.sh > /dev/null 2>&1`

in the log file--> gcloud command not found

I tried adding my path to the gcloud sdk in my case /home/user/google-cloud-sdk/bin/gcloud
to the $PATH variable and when I called `echo $PATH` in the terminal the directory was well in PATH.

What i figured is that crontab runs through a limited PATH environment including only /usr/bin and /bin. No matter how I tried to mention to crontab to take into account my sdk path, it just would not use it.

So What I did is really simple. I just created a symlink to gcloud in the /usr/bin folder this way

`$ sudo  ln -s /home/user/google-cloud-sdk/bin/gcloud /usr/bin/gcloud`

This completely fixed the issue. As long as the script for auto snapshot does not require any other command from the gcloud-sdk environment, it will work like a charm for most users I guess.

Hope I could help someone!

Cheers!
